### PR TITLE
Add FAQ regarding meta box breakage

### PR DESCRIPTION
**Pull request by kevinwhoffman**
## Description

This answer is paraphrased from a comment that Riad left in the Advanced WordPress Facebook Group. I also changed "metabox" to "meta box" throughout the FAQ for consistency with other official docs.

## Types of changes

Documentation

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -57,12 +57,20 @@ The API for creating blocks is a crucial aspect of the project. We are working o
...
+## Given Gutenberg is built in JavaScript, how will old meta boxes (PHP) work?
 
-We plan to continue supporting existing metaboxes while providing new ways to extend the interface.
+We plan to continue supporting existing meta boxes while providing new ways to extend the interface.
 
 *See:* [Pull request #2804](https://github.com/WordPress/gutenberg/pull/2804)
 
+## What are some reasons why an existing PHP meta box might not work in Gutenberg?
+
+Most PHP meta boxes should continue to work in Gutenberg, however some meta boxes that include advanced functionality could break. The following list describes some of the most common reasons why meta boxes might not work as expected in Gutenberg.
```
Comment (gziolo) [inline docs/faq.md]: It would be nice to have `:` instead of `.` at the end of the line.

File: docs/faq.md [hunk 2]
```diff
@@ -57,12 +57,20 @@ The API for creating blocks is a crucial aspect of the project. We are working o
...
 
-## Given Gutenberg is built in JavaScript, how will old metaboxes (PHP) work?
+## Given Gutenberg is built in JavaScript, how will old meta boxes (PHP) work?
 
-We plan to continue supporting existing metaboxes while providing new ways to extend the interface.
+We plan to continue supporting existing meta boxes while providing new ways to extend the interface.
 
 *See:* [Pull request #2804](https://github.com/WordPress/gutenberg/pull/2804)
 
+## What are some reasons why an existing PHP meta box might not work in Gutenberg?
```
Comment (youknowriad) [inline docs/faq.md]: do you think the `docs/meta-box.md` would be a better place for this?

Comment (kevinwhoffman) [reply docs/faq.md]: Yes, I'll move it there now.

**Review by youknowriad — APPROVED** [review]
👍

**Pull request merged**